app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,20 @@
   </head>
 
   <body>
-    <%= yield %>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-datetimepicker/2.5.4/build/jquery.datetimepicker.full.min.js"></script>
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery-datetimepicker/2.5.4/build/jquery.datetimepicker.min.css" />
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+    <div class="container">
+      <% flash.each do |key, value| %>
+        <div class="alert alert-<%= key %>">
```

Comment:
Kind of weird to have a separate flash div for each flash. It might be nicer to organize this as a list or something.
Something like
```
flash-div
  ul
    flash.each do 
      li - flash-msg
      button
      /li
    end
  /ul
/div
```